_posts/2021-04-13-upgrade-notification-for-ase.md

```diff
@@ -0,0 +1,83 @@
+3. In the left menu items, click **Alerts**.
+4. Click **Service Health**.
+5. Click **Add service health alert** at the top center.
+6. In the Condition section, choose the subscription that owns your ASEs.
+7. At the Service(s) box, choose all items starting with `App Service`:
+   * App Service
+   * App Service \ Web Apps
+   * App Service (Linux)
+   * App Service (Linux) \ Web App for Containers
+   * App Service (Linux) \ Web App.
+8. At the Region(s) box, make sure to check the regions of the ASEs.
+9. At the Event type box, check **Planned maintenance**.
+10. In the Actions section, click **Add action groups**.
+11. Click **Create alert rule**.
+12. Select the subscription that you have created your Logic app during the previous part.
+13. Choose a resource group and name an action group. Set Display name to something you can easily identify the action (**IMPORTANT**: The display name will be shown in every email/SMS/post of the notifications.)
+14. *If you want to receive text notifications*: In the Notifications section, choose **Email/SMS message/Push/Voice** at the Notification type. Then choose output channels you need (For example, Email and SMS.) Put email addresses or phone number as necessary.
+15. *If you want to hook up your custom automation*: In the Actions section, choose **Azure Function** or **Logic App** at the Action type. Put a name into the Name. Select your app.
+16. Press **Save changes**. The page will go back to the Rules management page.
+17. In the Alert rule details section, set a name.
+18. Click **Save**.
+
+## More resources
+
+* [Azure Monitor documentation](https://docs.microsoft.com/azure/azure-monitor/)
+* [Common alert schema definitions](https://docs.microsoft.com/en-us/azure/azure-monitor/alerts/alerts-common-schema-definitions#:~:text=Essentials%20%20%20%20Field%20%20%20,the%20alert%20...%20%209%20more%20rows%20)
+* [Logic App sample for automatic traffic diversion for Azure App Service](https://github.com/Azure-Samples/azure-logic-app-traffic-update-samples)
+
+## FAQ
+
+### When do you send the upgrade notifications?
+
+The first upgrade notifications will be created about 60 to 90 minutes before an actual upgrade operation starts. We don't create notifications anything earlier due to some limitations at this moment.
+
+Once the upgrade starts, we send in-progress notifications every 12 hours until the operation completes. After it's finished we send a notification of completion.
+
+### Can we get notifications earlier, like one day before?
+
+No. At this point from 60 minutes to 90 minutes is the earliest timing of upgrade notifications.
+
+### Is it only available for ASEs?
+
+Yes, currently it is only available for ASEs. We are actively working on enabling it for all public Azure.
+
+### Can we invoke my Azure Function when a notification comes?
```

Comment:
Can I instead of we (or use our instead of my) 😊

Reply:
Updated